### PR TITLE
Refactor/material dialogs

### PR DESCRIPTION
**Pull request by fchauveau**
Hey,
I just made all dialogs button as material.
I also did some refactor in building dialogs.
This could lead to some lose of style in few cases

Have fun reading PR  ;)

### CODE REVIEW

File: library/src/main/java/com/codetroopers/betterpickers/datepicker/DatePickerDialogFragment.java [hunk 1]
```diff
@@ -101,76 +97,58 @@ public void onCreate(Bundle savedInstanceState) {
...
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-            Bundle savedInstanceState) {
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.date_picker_dialog, null);
 
-        View v = inflater.inflate(R.layout.date_picker_dialog, null);
-        mSet = (Button) v.findViewById(R.id.set_button);
-        mCancel = (Button) v.findViewById(R.id.cancel_button);
-        mCancel.setOnClickListener(new View.OnClickListener() {
+        Button mSetButton = (Button) view.findViewById(R.id.done_button);
```
Comment (CedricGatay) [inline library/src/main/java/com/codetroopers/betterpickers/datepicker/DatePickerDialogFragment.java]: I think we should drop the m as it is not a member anymore, same applies to cancel

File: library/src/main/java/com/codetroopers/betterpickers/expirationpicker/ExpirationPickerDialogFragment.java [hunk 2]
```diff
@@ -101,48 +97,35 @@ public void onCreate(Bundle savedInstanceState) {
 
         // Init defaults
         mTextColor = getResources().getColorStateList(R.color.dialog_text_color_holo_dark);
-        mButtonBackgroundResId = R.drawable.button_background_dark;
-        mDividerColor = getResources().getColor(R.color.default_divider_color_dark);
+        int mButtonBackgroundResId = R.drawable.button_background_dark;
```
Comment (CedricGatay) [inline library/src/main/java/com/codetroopers/betterpickers/expirationpicker/ExpirationPickerDialogFragment.java]: wrong prefix 'm'

File: library/src/main/java/com/codetroopers/betterpickers/expirationpicker/ExpirationPickerDialogFragment.java [hunk 3]
```diff
@@ -101,48 +97,35 @@ public void onCreate(Bundle savedInstanceState) {
...
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
-            Bundle savedInstanceState) {
+                             Bundle savedInstanceState) {
 
-        View v = inflater.inflate(R.layout.expiration_picker_dialog, null);
-        mSet = (Button) v.findViewById(R.id.set_button);
-        mCancel = (Button) v.findViewById(R.id.cancel_button);
-        mCancel.setOnClickListener(new View.OnClickListener() {
+        View view = inflater.inflate(R.layout.expiration_picker_dialog, null);
+        Button mSetButton = (Button) view.findViewById(R.id.done_button);
```
Comment (CedricGatay) [inline library/src/main/java/com/codetroopers/betterpickers/expirationpicker/ExpirationPickerDialogFragment.java]: wrong prefix 'm'

File: library/src/main/java/com/codetroopers/betterpickers/hmspicker/HmsPickerDialogFragment.java [hunk 4]
```diff
@@ -75,40 +71,34 @@ public void onCreate(Bundle savedInstanceState) {
...
-            Bundle savedInstanceState) {
+                             Bundle savedInstanceState) {
 
-        View v = inflater.inflate(R.layout.hms_picker_dialog, null);
-        mSet = (Button) v.findViewById(R.id.set_button);
-        mCancel = (Button) v.findViewById(R.id.cancel_button);
-        mCancel.setOnClickListener(new View.OnClickListener() {
+        View view = inflater.inflate(R.layout.hms_picker_dialog, null);
+
+        Button mSetButton = (Button) view.findViewById(R.id.done_button);
```
Comment (CedricGatay) [inline library/src/main/java/com/codetroopers/betterpickers/hmspicker/HmsPickerDialogFragment.java]: wrong prefix 'm'

File: library/src/main/java/com/codetroopers/betterpickers/numberpicker/NumberPickerDialogFragment.java [hunk 5]
```diff
@@ -146,39 +142,33 @@ public void onCreate(Bundle savedInstanceState) {
...
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-            Bundle savedInstanceState) {
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
 
-        View v = inflater.inflate(R.layout.number_picker_dialog, null);
-        mSet = (Button) v.findViewById(R.id.set_button);
-        mCancel = (Button) v.findViewById(R.id.cancel_button);
-        mCancel.setOnClickListener(new View.OnClickListener() {
+        View view = inflater.inflate(R.layout.number_picker_dialog, null);
+        Button mSetButton = (Button) view.findViewById(R.id.done_button);
```
Comment (CedricGatay) [inline library/src/main/java/com/codetroopers/betterpickers/numberpicker/NumberPickerDialogFragment.java]: wrong prefix 'm'

File: library/src/main/java/com/codetroopers/betterpickers/timepicker/TimePickerDialogFragment.java [hunk 6]
```diff
@@ -72,40 +68,32 @@ public void onCreate(Bundle savedInstanceState) {
...
+            mDialogBackgroundResId = a.getResourceId(R.styleable.BetterPickersDialogFragment_bpDialogBackground, mDialogBackgroundResId);
         }
     }
 
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-            Bundle savedInstanceState) {
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.time_picker_dialog, null);
+        Button mSetButton = (Button) view.findViewById(R.id.done_button);
```
Comment (CedricGatay) [inline library/src/main/java/com/codetroopers/betterpickers/timepicker/TimePickerDialogFragment.java]: wrong prefix 'm'

**Comment by CedricGatay** [comment]
Apart from a few naming conventions, looks legit to me.

**Comment by fchauveau** [comment]
merging, going to release soon

**Pull request merged**